src/pg_to_evalscript/javascript_processes/order.js

```diff
@@ -0,0 +1,63 @@
+function order(arguments) {
+  const { data, asc = true, nodata = null} = arguments;
+
+  validateParameter({
+    processName: "order",
+    parameterName: "data",
+    value: data,
+    required: true,
+    array: true,
+  });
+
+  validateParameter({
+    processName: "order",
+    parameterName: "asc",
+    value: asc,
+    required: true,
```

Comment:
This `required: true` might be misleading since the parameter is not actually required

Reply:
Hm, this was still functioning as expected because default value is set. Nevertheless, it should be updated as I agree it's misleading.